packages/headlessui-float-vue/src/resolver.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore
```

Comment:
Hi! can you change to `/** @ts-ignore */`?

Reply:
Sure, done!